src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -22,15 +42,28 @@ describe('UserManagementComponent', () => {
         children: [],
     } as any as ActivatedRoute;
 
-    beforeEach(async () => {
+    beforeEach(() => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, ArtemisAdminModule, RouterTestingModule],
+            imports: [MockModule(ReactiveFormsModule), MockModule(NgbModule), HttpClientTestingModule, MockModule(RouterTestingModule)],
```

Comment:
I do not see any occurence of the HttpTestingModule in the file, and maybe the RouterTestingModule could be replaced by simpling using the MockRouter since it also does not seem to be used. Or I overlooked it.

Reply:
you are right, when I add `{ provide: Router, useClass: MockRouter },` I do not need `HttpClientTestingModule, MockModule(RouterTestingModule)`. Even better!